### PR TITLE
Added JBA PE branches

### PR DESCRIPTION
**Pull request by xgui3783**
feat: add JBA PE branches
feat: updated JBA PE leave parent attr

### CODE REVIEW

**Comment by xgui3783** [comment]
@lzehl seems like I have no write access to hbp/openminds_sands, so I had to push to my own fork and raise PR

additionally, I cannot set you as reviewer. but can you review nontheless?

**Comment by lzehl** [comment]
@xgui3783 thank you for the PR. Yes I'll review it. You indeed had to go the external contributors way. I can imagine though adding you to the developer list for SANDS you may get involved a lot I imagine ;)

**Comment by xgui3783** [comment]
thought I would imagine having an external contributor perspective might also be helpful.

Looking forward to the review. 

also something strikes me: would it be useful to have a visualizer for the various `hasParent` attributes?

File: instances/atlas/parcellationEntity/JBA/JBA_cingulate-gyrus--frontal-part.jsonld [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_cingulate-gyrus--frontal-part",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_cingulate-gyrus--frontal-part.jsonld]: special signs such as "," are typically replaced by "_" in most namings we use

File: instances/atlas/parcellationEntity/JBA/JBA_cingulate-gyrus--frontal-part.jsonld [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_limbic-lobe"
+        }
+    ],
+    "hasVersion": null,
+    "lookupLabel": "JBA_cingulate-gyrus--frontal-part",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_cingulate-gyrus--frontal-part.jsonld]: special signs such as "," are typically replaced by "_" in most namings we use

File: instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_overall",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld]: this seems not correct... do you mean "brain" ?

Comment (lzehl) [reply instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld]: or Julich-Brain ?

File: instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_overall"
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld]: cannot be the parent of itself. if it is the highest parent it should just have no parent

File: instances/atlas/parcellationEntity/JBA/JBA_temporal-insula.jsonld [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_temporal-insula",
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_temporal-lobe"
+        }
+    ],
+    "hasVersion": null,
+    "lookupLabel": "JBA_temporal-insula",
+    "name": "temporal-insula",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_temporal-insula.jsonld:14]: I think the name should be without "-", no?

File: instances/atlas/parcellationEntity/JBA/JBA_Area-25.jsonld [hunk 6]
```diff
@@ -4,7 +4,11 @@
     },
     "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-25",
     "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_cingulate-gyrus--frontal-part"
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-25.jsonld]: the "--" should be a "_" (see comment for parent)

File: instances/atlas/parcellationEntity/JBA/JBA_Area-33.jsonld [hunk 7]
```diff
@@ -4,7 +4,11 @@
     },
     "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-33",
     "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_cingulate-gyrus--frontal-part"
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-33.jsonld]: the "--" should be a "_" (see comment for parent)

File: instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld [hunk 8]
```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-Op5",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld]: this should be OP5 (as it was OP1 - OP4)

File: instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_frontal-operculum"
+        }
+    ],
+    "hasVersion": null,
+    "lookupLabel": "JBA_Area-Op5",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld]: this should be OP5 (as it was OP1 - OP4)

File: instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+    "name": "Area Op5 (Frontal Operculum)",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld]: this should be OP5 (as it was OP1 - OP4)

File: instances/atlas/parcellationEntity/JBA/JBA_Area-Op6.jsonld [hunk 11]
```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-Op6",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-Op6.jsonld]: this should be OP6 (as it was OP1 - OP4) also below for lookupLabel and name

File: instances/atlas/parcellationEntity/JBA/JBA_Area-Op7.jsonld [hunk 12]
```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-Op7",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-Op7.jsonld]: this should be OP7 (as it was OP1 - OP4) also below for lookupLabel and name

File: instances/atlas/parcellationEntity/JBA/JBA_Area-Op8.jsonld [hunk 13]
```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-Op8",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-Op8.jsonld]: this should be OP8 (as it was OP1 - OP4) also below for lookupLabel and name

File: instances/atlas/parcellationEntity/JBA/JBA_Area-Op9.jsonld [hunk 14]
```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-Op9",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-Op9.jsonld]: this should be OP9 (as it was OP1 - OP4) also below for lookupLabel and name

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-0.jsonld [hunk 15]
```diff
@@ -2,9 +2,13 @@
     "@context": {
         "@vocab": "https://openminds.ebrains.eu/vocab/"
     },
-    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-1.0",
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-1-0",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-0.jsonld]: you can leave the "." in the "@id" this is fine.

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-0.jsonld [hunk 16]
```diff
@@ -2,9 +2,13 @@
...
     "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Heschl-s-gyrus"
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-0.jsonld]: I would delete the "-" between Heschl and s

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-0.jsonld [hunk 17]
```diff
@@ -64,7 +68,7 @@
             "@id": "https://openminds.ebrains.eu/instances/JBA-v2.4_MNI-ICBM-152-2009c-nonlin-asym_Area-TE-1.0_PM-v5.1_right"
         }
     ],
-    "lookupLabel": "JBA_Area-TE-1.0",
+    "lookupLabel": "JBA_Area-TE-1-0",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-0.jsonld]: also here: the "." is okay

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-1.jsonld [hunk 18]
```diff
@@ -2,9 +2,13 @@
     "@context": {
         "@vocab": "https://openminds.ebrains.eu/vocab/"
     },
-    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-1.1",
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-1-1",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-1.jsonld]: leave the "."

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-1.jsonld [hunk 19]
```diff
@@ -64,7 +68,7 @@
             "@id": "https://openminds.ebrains.eu/instances/JBA-v1.18_Colin27-1998_Area-TE-1.1_PM-v5.1_left"
         }
     ],
-    "lookupLabel": "JBA_Area-TE-1.1",
+    "lookupLabel": "JBA_Area-TE-1-1",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-1.jsonld]: leave the "."

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-2.jsonld [hunk 20]
```diff
@@ -2,9 +2,13 @@
     "@context": {
         "@vocab": "https://openminds.ebrains.eu/vocab/"
     },
-    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-1.2",
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-1-2",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-2.jsonld]: leave the "."

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-2.jsonld [hunk 21]
```diff
@@ -64,7 +68,7 @@
             "@id": "https://openminds.ebrains.eu/instances/JBA-v2.5_MNI-ICBM-152-2009c-nonlin-asym_Area-TE-1.2_PM-v6.0_right"
         }
     ],
-    "lookupLabel": "JBA_Area-TE-1.2",
+    "lookupLabel": "JBA_Area-TE-1-2",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-2.jsonld]: leave the "."

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-2-1.jsonld [hunk 22]
```diff
@@ -2,9 +2,13 @@
     "@context": {
         "@vocab": "https://openminds.ebrains.eu/vocab/"
     },
-    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-2.1",
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-2-1",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-TE-2-1.jsonld]: leave the "."

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-2-1.jsonld [hunk 23]
```diff
@@ -49,7 +53,7 @@
             "@id": "https://openminds.ebrains.eu/instances/JBA-v2.4_MNI-ICBM-152-2009c-nonlin-asym_Area-TE-2.1_PM-v5.1_right"
         }
     ],
-    "lookupLabel": "JBA_Area-TE-2.1",
+    "lookupLabel": "JBA_Area-TE-2-1",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-TE-2-1.jsonld]: leave the "."

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-2-2.jsonld [hunk 24]
```diff
@@ -2,9 +2,13 @@
     "@context": {
         "@vocab": "https://openminds.ebrains.eu/vocab/"
     },
-    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-2.2",
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-2-2",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-TE-2-2.jsonld]: leave the "."

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-2-2.jsonld [hunk 25]
```diff
@@ -49,7 +53,7 @@
             "@id": "https://openminds.ebrains.eu/instances/JBA-v2.5_MNI-ICBM-152-2009c-nonlin-asym_Area-TE-2.2_PM-v6.0_right"
         }
     ],
-    "lookupLabel": "JBA_Area-TE-2.2",
+    "lookupLabel": "JBA_Area-TE-2-2",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-TE-2-2.jsonld]: leave the "."

File: instances/atlas/parcellationEntity/JBA/JBA_Area-p24ab.jsonld [hunk 26]
```diff
@@ -4,7 +4,11 @@
     },
     "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-p24ab",
     "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_cingulate-gyrus--frontal-part"
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-p24ab.jsonld]: "--" should be "_" see parent comment

File: instances/atlas/parcellationEntity/JBA/JBA_Area-p24c.jsonld [hunk 27]
```diff
@@ -4,7 +4,11 @@
     },
     "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-p24c",
     "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_cingulate-gyrus--frontal-part"
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-p24c.jsonld]: "--" should be "_" see parent comment

File: instances/atlas/parcellationEntity/JBA/JBA_Area-p32.jsonld [hunk 28]
```diff
@@ -4,7 +4,11 @@
     },
     "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-p32",
     "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_cingulate-gyrus--frontal-part"
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-p32.jsonld]: "--" should be "_" see parent comment

File: instances/atlas/parcellationEntity/JBA/JBA_Area-s24.jsonld [hunk 29]
```diff
@@ -4,7 +4,11 @@
     },
     "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-s24",
     "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_cingulate-gyrus--frontal-part"
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-s24.jsonld]: "--" should be "_" see parent comment

File: instances/atlas/parcellationEntity/JBA/JBA_Area-s32.jsonld [hunk 30]
```diff
@@ -4,7 +4,11 @@
     },
     "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-s32",
     "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_cingulate-gyrus--frontal-part"
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Area-s32.jsonld]: "--" should be "_" see parent comment

File: instances/atlas/parcellationEntity/JBA/JBA_Heschl-s-gyrus.jsonld [hunk 31]
```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Heschl-s-gyrus",
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_Heschl-s-gyrus.jsonld]: I would use "JBA_Heschls-gyrus" (but this is up to you)

File: instances/atlas/parcellationEntity/JBA/JBA_metencephalon.jsonld [hunk 32]
```diff
@@ -0,0 +1,13 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_metencephalon",
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": null,
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_metencephalon.jsonld]: missing parent

File: instances/atlas/parcellationEntity/JBA/JBA_telencephalon.jsonld [hunk 33]
```diff
@@ -0,0 +1,13 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_telencephalon",
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": null,
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_telencephalon.jsonld]: missing parent

File: instances/atlas/parcellationEntity/JBA/JBA_diencephalon.jsonld [hunk 34]
```diff
@@ -0,0 +1,13 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_diencephalon",
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": null,
```
Comment (lzehl) [inline instances/atlas/parcellationEntity/JBA/JBA_diencephalon.jsonld]: missing parent

**Review by lzehl — CHANGES_REQUESTED** [review]
@xgui3783 thank you again for this PR and the support. I've added review comments (feel free to get back to me if there are questions or you need to discuss something) 

Also here some general comments/issues:

All jsonld miss an additional line at the end of each file (this is a something that github complains about and is an "error" in json.dump function in python...)

This is just a note for future cases. I would leave it here as it is, but typically we use a lowerCamelCase for the "@id" and the lookup labels. I did not do this as well for the Areas, because it would have messed up the correct writing of some of the labels. 

"JBA_overall" should be renamed to "JBA_brain" and should also be the parent of "telencephalon", "diencephalon" and "metencephalon".

File: instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld (hunk 3 above)
Comment (xgui3783) [reply instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld]: This is an error.

It seems in our definition there is a brain region called `overall`, (which serves as the root structure). 

do you think it's suitable to remove this instance, and have its children have no parents? 

(taken out of context the last line reads a little dark)

File: instances/atlas/parcellationEntity/JBA/JBA_temporal-insula.jsonld (hunk 5 above)
Comment (xgui3783) [reply instances/atlas/parcellationEntity/JBA/JBA_temporal-insula.jsonld:14]: It is as the name in the json hartmut gave us

https://gin.g-node.org/FZJ-INM1-BDA/20210615-julichbrain-v2.9.0/src/master/workingdir/hbp-d000006_julichbrain-cytoatlas_dev/JulichBrainCytoAtlas/julichbrain-ontology_29.0.json

(require authentication/authorization, what is your gin user name? I will add you?)

we can certainly change it, just say the word.

File: instances/atlas/parcellationEntity/JBA/JBA_metencephalon.jsonld (hunk 32 above)
Comment (xgui3783) [reply instances/atlas/parcellationEntity/JBA/JBA_metencephalon.jsonld]: In our hierarchy, metencephalon (& telencephalon) are root structures. 

What would the parents be?

File: instances/atlas/parcellationEntity/JBA/JBA_cingulate-gyrus--frontal-part.jsonld (hunk 1 above)
Comment (xgui3783) [reply instances/atlas/parcellationEntity/JBA/JBA_cingulate-gyrus--frontal-part.jsonld]: ~~I don't see `,` ? do you mean `--` ?~~

edit: I think I see what you mean. replace `,` with `_`, and space with `-`

Comment (lzehl) [reply instances/atlas/parcellationEntity/JBA/JBA_cingulate-gyrus--frontal-part.jsonld]: The name of "JBA_cingulate-gyrus--frontal-part" is "cingulate-gyrus, frontal-part".
Typically we replace special signs with "_" in the "@id" and file names. E.g.  "JBA_cingulate-gyrus_frontal-part"

But: I realized that in all other cases I think in the JBA uses the convention "frontal ...", "dorsal ..." etc. 
Should we just rename this completely anyways? To "JBA_frontal-cingulate-gyrus"?

File: instances/atlas/parcellationEntity/JBA/JBA_metencephalon.jsonld (hunk 32 above)
Comment (lzehl) [reply instances/atlas/parcellationEntity/JBA/JBA_metencephalon.jsonld]: as stated in the general comment: The brain is the root structure. Therefore I would rename JBA_overall to JBA_brain and then group all GabMaps and telencephalon, metencephalon and diencephalon below it.

File: instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld (hunk 3 above)
Comment (lzehl) [reply instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld]: I would rename it to JBA_brain and use it as root structure for all GapMaps as well as telencephalon, metencephalon and diencephalon.

File: instances/atlas/parcellationEntity/JBA/JBA_temporal-insula.jsonld (hunk 5 above)
Comment (lzehl) [reply instances/atlas/parcellationEntity/JBA/JBA_temporal-insula.jsonld:14]: well if this is what he defines leave it for now. we can discuss refinements with the Julich-Brain team later.

File: instances/atlas/parcellationEntity/JBA/JBA_cingulate-gyrus--frontal-part.jsonld (hunk 1 above)
Comment (xgui3783) [reply instances/atlas/parcellationEntity/JBA/JBA_cingulate-gyrus--frontal-part.jsonld]: so `JBA_{position_quantifier}-{name}` ?

I can do that

Comment (lzehl) [reply instances/atlas/parcellationEntity/JBA/JBA_cingulate-gyrus--frontal-part.jsonld]: Yes. I think this is done in nearly all cases (e.g., "lateral occipital cortex", "frontal pole", etc)

File: instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld (hunk 8 above)
Comment (xgui3783) [reply instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld]: this also came directly from the json file hartmut passed us:

https://gin.g-node.org/FZJ-INM1-BDA/20210615-julichbrain-v2.9.0/src/master/workingdir/hbp-d000006_julichbrain-cytoatlas_dev/JulichBrainCytoAtlas/julichbrain-ontology_29.0.json#L1149

we can make the change now, if you give the OK.

File: instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld (hunk 9 above)
Comment (xgui3783) [reply instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld:13]: please see https://github.com/HumanBrainProject/openMINDS_SANDS/pull/151#discussion_r765939025

File: instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld (hunk 10 above)
Comment (xgui3783) [reply instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld:14]: please see https://github.com/HumanBrainProject/openMINDS_SANDS/pull/151#discussion_r765939025

File: instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-0.jsonld (hunk 16 above)
Comment (xgui3783) [reply instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-0.jsonld]: so a general rule is to remove all `'`s ?

File: instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld (hunk 3 above)
Comment (xgui3783) [reply instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld]: I doubled checked (https://gin.g-node.org/FZJ-INM1-BDA/20210615-julichbrain-v2.9.0/src/master/workingdir/hbp-d000006_julichbrain-cytoatlas_dev/JulichBrainCytoAtlas/julichbrain-ontology_29.0.json#L1728-L1740)

hartmut seems to use (nested) overal as super structure for the gapmaps (only) and nothing else. (In the json hierarchy there, it was suppose to be a child of cerebral cortex. Why in this case we have overall being parent of itself is a glitch (I use region name to generate id, and output jsonld name. so two regions with the same name meant one will overwrite the other)

I can certainly make adjustments, but I would like a few clarifications:

- I can certainly make a root region called `Julich-Brain` (independent of this `overall` branch node)
- how (if we should at all) change the branch nodes named `overall` (see screengrab below. or you could explore it at https://atlases.ebrains.eu/viewer/#/a:juelich:iav:atlas:v1.0.0:1/t:minds:core:referencespace:v1.0.0:dafcffc5-4826-4bf1-8ff6-46b8a31ff8e2/p:minds:core:parcellationatlas:v1.0.0:94c1125b-b87e-45e4-901c-00daee7f2579-290/@:0.0.0.-W000.._eCwg.2-FUe3._-s_W.2_evlu..7LIx..0.0.0..1LSm)
Screenshot from 2021-12-09 17-33-59

File: instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld (hunk 4 above)
Comment (xgui3783) [reply instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld]: related to https://github.com/HumanBrainProject/openMINDS_SANDS/pull/151#discussion_r765961546

**Comment by xgui3783** [comment]
discussed with @lzehl , here is what we decided:
- remove `overall`'s
- move all gapmaps under cerebral cortex
- capitalize all `Op` -> `OP`
- add Julich Brain as root structure

**Comment by xgui3783** [comment]
@lzehl can you take a look at the PR again?

For the root structure, I added https://github.com/xgui3783/SANDS/blob/feat_addJulichBrainPE/instances/atlas/parcellationEntity/JBA/JBA_julich-brain.jsonld

and I left https://github.com/HumanBrainProject/openMINDS_SANDS/pull/151#discussion_r765636714 unresolved, not sure if we would like to record somewhere to discuss naming of `temporal-insula`

**Review by lzehl — APPROVED** [review]

**Comment by lzehl** [comment]
@xgui3783 a big big thank you for this one ;)

**Pull request merged**